core-bundle/src/Resources/views/Frontend/preview_toolbar_base.html.twig

```diff
@@ -2,9 +2,12 @@
 <div class="cto-toolbar__open">
     <a href="#" title="{{ 'MSC.openToolbar'|trans }}">{{ include('@ContaoCore/Collector/contao.svg') }}</a>
 </div>
-<div class="cto-toolbar__inside">
+<div class="cto-toolbar__inside" {{ attributes|raw }}>
```

Comment:
Without a helper function and passing the associative array, this would become something like that:
```suggestion
{% set attributes = attributes|merge({class: 'cto-toolbar__open' }) %}
<div {{ attributes|map((v, k) => '%s="%s"'|format(k,v))|join(' ')|raw }}>
```

If we do not trust the input are safe attributes we could even do this, i.e. escaping the attribute parts (btw., this is exactly why the template should compose, not the code. :slightly_smiling_face:):
```twig
attributes|map((v, k) => '%s="%s"'|format(k|e('html_attr'), v|e('html')))|join(' ')|raw
```
/cc @ausi

Reply:
I think I would prefer something like this:

```suggestion
<div
    class="cto-toolbar__inside"
    {% for key, value in attributes %}
        {{ key|html_attr }}="{{ value }}"
    {% endfor %}
>
```